app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -165,7 +165,7 @@ class BrowserToolbarIntegration(
                 menuItemIndex = menu.menuBuilder.items.size - 1,
                 closeListener = { fragment.closeCustomTab() },
                 updateToolbarBackground = true,
-                forceActionButtonTinting = false,
```

Comment:
I think the team wanted to keep trailing commas around. We probably want to enable these [two rules](https://pinterest.github.io/ktlint/rules/standard/#trailing-comma-on-call-site)?

Reply:
Unfortunately, running ktlint auto format would mean either removing all existing trailing commas or adding trailing commas everywhere. The alternative would be manually modifying a serious amount of files.

I searched for a style guide or a decision in this issue but did not find one. IMO we should add trailing commas everywhere :)

Reply:
We could use a baseline file for ktlint but using ktlintFormat to update them all once isn't so bad either.

Reply:
I added a baseline file, I think it is a better option than having a commit with 80 + modified files.
Also tried for AC, there would have been 200+ modified files. IMO we should stick to the same approach in all 3 repos.
And maybe improve the existing issues only when that code is touched. Thoughts?

Reply:
> And maybe improve the existing issues only when that code is touched. Thoughts?

Yeah, I'm fine with that. 

In general, I'm usually more flexible with Focus because it's a smaller project (code base) so we can try out experiments like this with ease.